subiquity/server/controllers/install.py

```diff
@@ -688,6 +688,7 @@ async def postinstall(self, *, context):
                     hostname = f.read().strip()
 
             await self.app.controllers.Ad.join_domain(hostname, context)
+        self.model.filesystem.copy_artifacts_to_target()
```

Comment:
user hasn't been created yet

Reply:
but the key is getting copied to /var/log/installer right?

Reply:
Yes, it's exposed in $HOME in the live session, but then ends up copied to /var/log/installer in the target.